tools/image_processing/yolo/doclayoutyolo.xml

```diff
@@ -0,0 +1,124 @@
+<tool id="doclayout_yolo" name="DocLayout-YOLO: Enhancing Document Layout Analysis" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
+    <description>through Diverse Synthetic Data and Global-to-Local Adaptive Perception</description>
```

Comment:
Isn't that description too detailed. Maybe the name simply "doclayout-yolo" and the description "enhancing document layout analysis"? The current description maybe move to the help?

Reply:
Ok, done!